sqlQueries/nlstCohort.sql

```diff
@@ -138,7 +146,8 @@ geometryChecks AS (
       -- add " idc_data/" which acts as destination folder for s5cmd and
       -- at end for each row separated by new line character "\n" so it will be
       -- ready for creating a manifest file later in the workflow
-      string_agg (CONCAT("cp ",REPLACE(sopInstanceUrl, "gs://", "s3://"), " idc_data/"), "\n") as s5cmdUrls
+      --string_agg (CONCAT("cp ",REPLACE(sopInstanceUrl, "gs://", "s3://"), " idc_data/"), "\n") as s5cmdUrls
+       ANY_VALUE(CONCAT("cp --show-progress s3",REGEXP_SUBSTR(sopInstanceUrl, "(://.*)/"),"/* idc_data/")) AS s5cmdUrls
```

Comment:
Instead of using this, let's use `aws_bucket` + `crdc_series_uuid`, and we don't need to aggregate to get it!

Reply:
Sure